src/init.cpp

```diff
@@ -752,13 +753,19 @@ void InitParameterInteraction()
 
     // disable walletbroadcast and whitelistalwaysrelay in blocksonly mode
```

Comment:
You'd need to make the rename globally; Also, I don't like that this just ignores    `-whitelistalwaysrelay` set in old `.conf` files.